components/Dialogs/src/MDCDialogTransitionController.h

```diff
@@ -34,4 +34,19 @@
 @interface MDCDialogTransitionController
     : NSObject <UIViewControllerAnimatedTransitioning, UIViewControllerTransitioningDelegate>
 
+/**
+ Duration of transition animation.
+
+ Defaults to 0.27.
```

Comment:
Tip: Please document this with units. For example, "Defaults to 270 milliseconds."